### DISCUSSION TITLE
How to standardize post urls?

### DISCUSSION THREAD

**Discussion by zhhbinn** [GetPublii/Publii#1450] [Help]
How can I manage articles under a specific directory uniformly? Currently, the document links are structured as HOST/post-name, but I am looking to standardize them to HOST/a/post-name while maintaining the homepage at HOST/. Could you please advise me on how to achieve this structure? Thank you.

**Comment by bobmitro** [comment]
It's not possible right now, but....:). Here is how Publii handles the URL: https://getpublii.com/dev/how-urls-in-publii-works/

**Reply by zhhbinn** [reply]
Thank you for your response. I am really looking forward to your 'but'——the turning point.

**Comment by candidexmedia** [comment]
Related discussion: https://github.com/GetPublii/Publii/discussions/1413
